LibraBFT/Impl/Properties/VotesOnce.agda

```diff
@@ -185,13 +188,13 @@ module LibraBFT.Impl.Properties.VotesOnce where
                     -- So we have proved both that the round of v' is ≤ the lastVotedRound of
                     -- the peer's state and that the round of v' is one greater than that value,
                     -- which leads to a contradiction
-                    = ⊥-elim (1+n≰n (≤-trans (≤-reflexive suclvr≡v'rnd)
-                                             (≤-trans (≤-reflexive rnds≡) v'rnd≤lvr)))
+                    = inj₁ {!!} --⊥-elim (1+n≰n (≤-trans (≤-reflexive suclvr≡v'rnd)
```

Comment:
I have completed this proof in another branch, so I will add it after you merge.